### PR TITLE
Fix small GLEC ID issues

### PR DESCRIPTION
**Pull request by patstirling**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Road Freight

**What?**

Several small fixes to IDs

**Why?**

These were erroneous

### CODE REVIEW

**Review by maryarbab — CHANGES_REQUESTED** [review]
Looks good to me, @patstirling! Just two notes:
1- "t" seems to be missing in the lower range in rows 19637-19640
2- I assume these EFs are added to the dataset from page 104, right? So, the description needs to be Well-to-Wheel (WTW) for rows 19633-19640 and 19830-19833

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -19630,14 +19630,14 @@ a9607bb4-2bc0-48eb-afc5-ffaaf6d83390,Transport,Road Freight,freight_vehicle-vehi
...
+3cd3e74a-c6f0-4b5b-b2a4-d86c6b6a1c57,Transport,Road Freight,freight_vehicle-vehicle_type_mgv-fuel_source_na-vehicle_weight_gt_7.5_lt_20t-distance_uplift_included,MGV (7.5-20 t),tonne-km,not-supplied,0.244,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,,well_to_wheel,GLEC,,2019,2019,,ASIA_AFRICA,m,Climatiq,2022/06/20,Well-to-Tank (WTT) emission intensity of freight. Retrieved from the Global Logistics Emissions Council Framework for Logistics Emissions Accounting and Reporting. (2019). NOTE: The source does not clarify if the kgCO2e value is calculated using either IPCC Fourth Assessment Report (AR4) or IPCC Fifth Assessment Report (AR5) methodologies.This factor includes a +5% distance adjustment to allow for minor deviations and delays to the planned route.,https://www.smartfreightcentre.org/en/how-to-implement-items/what-is-glec-framework/58/
```
[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:19637]: Good catches - have added in the missing "t"s.

I hadn't reviewed the rest of the fields - but good spot -the description didn't reflect what was in the lca_activity field for these EFs, have corrected.

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -19630,14 +19630,14 @@ a9607bb4-2bc0-48eb-afc5-ffaaf6d83390,Transport,Road Freight,freight_vehicle-vehi
...
+6a0d821a-7c3b-4576-bfa9-e69439b5ada9,Transport,Road Freight,freight_vehicle-vehicle_type_mgv-fuel_source_na-vehicle_weight_gt_7.5t_lt_20t-distance_uplift_included,MGV (7.5-20 t),tonne-km,not-supplied,0.2,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,,well_to_wheel,GLEC,,2019,2019,,EU_S_AMERICA,m,Climatiq,2022/06/20,Well-to-Wheel (WT@) emission intensity of freight. Retrieved from the Global Logistics Emissions Council Framework for Logistics Emissions Accounting and Reporting. (2019). NOTE: The source does not clarify if the kgCO2e value is calculated using either IPCC Fourth Assessment Report (AR4) or IPCC Fifth Assessment Report (AR5) methodologies.This factor includes a +5% distance adjustment to allow for minor deviations and delays to the planned route.,https://www.smartfreightcentre.org/en/how-to-implement-items/what-is-glec-framework/58/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:19638]: There is a small typo here: WT@-> WTW

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:19638]: Fixed! Thanks

**Review by maryarbab — APPROVED** [review]

**Pull request merged**